katas/content/complex_arithmetic/complex_addition/Placeholder.qs

```diff
@@ -0,0 +1,14 @@
+namespace Kata {
+    
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Intrinsic;
+
+    @EntryPoint()
+    operation ComplexAdd(x : Complex, y : Complex) : Complex {
+    
```

Comment:
For spacing, we don't leave empty space after operation signature or after namespace declaration. Especially on the site, we get very few lines of code in the task/demo code windows, so making the code compact helps